### PR TITLE
PPM Closeout: Add customer agreement + legalese page

### PR DESCRIPTION
**Pull request by mkrump**
## Description

Add customer agreement and legalese page and submit buttons

## Reviewer Notes
Go to `/ppm-payment-review`, at the bottom you'll see the customer agreement checkbox. Clicking the link will take you to the `ppm-customer-agreement` page with the relevant legalese. Clicking back will take you back to the review page.

~~_The story styling the review page is still being worked on, so for now I've just added a placeholder with the relevant links, so you can ignore the body review page_~~

## Setup

```sh
make client_test
make e2e_test
```

## Code Review Verification Steps
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/n/projects/2181745/stories/166314473) for this change

## Screenshots
Screen Shot 2019-06-26 at 3 19 03 PM
Screen Shot 2019-06-25 at 8 32 50 AM
Screen Shot 2019-06-26 at 3 19 10 PM

### CODE REVIEW

File: src/scenes/Legalese/legaleseText.js [hunk 1]
```diff
@@ -89,3 +89,5 @@ have to remit the difference with the balance of my incentive disbursement and/o
 be necessary. If I receive an advance for my PPM shipment, I agree to furnish weight tickets within 45 days of final \
 delivery to my destination. I understand that failure to furnish weight tickets within this timeframe may lead to the \
 collection of my pay as necessary to cover the cost of the advance.`;
+
+// export const ppmPaymentLegalStyled =
```
Comment (mkrump) [inline src/scenes/Legalese/legaleseText.js]: remove

File: src/scenes/Moves/Ppm/PaymentReview/index.jsx [hunk 2]
```diff
@@ -4,12 +4,24 @@ import { ProgressTimeline, ProgressTimelineStep } from 'shared/ProgressTimeline'
...
+  };
+  handleOnAcceptTermsChange = acceptTerms => {
+    this.setState({ acceptTerms });
+  };
   render() {
+    const { submitting } = this.props;
     const moveId = this.props.match.params.moveId;
     const weightTicketsPage = `/moves/${moveId}/ppm-weight-ticket`;
     const expensePage = `/moves/${moveId}/ppm-expenses`;
+    const nextBtnLabel = 'Submit Request';
```
Comment (mkrump) [inline src/scenes/Moves/Ppm/PaymentReview/index.jsx]: pull out of class like other components.

File: src/scenes/Moves/Ppm/PaymentReview/index.jsx [hunk 3]
```diff
@@ -22,19 +35,42 @@ class PaymentReview extends Component {
...
+              link="/ppm-customer-agreement"
+              checked={this.state.acceptTerms}
+              agreementText={ppmPaymentLegal}
+            />
+          </div>
+          <PPMPaymentRequestActionBtns
+            nextBtnLabel={nextBtnLabel}
+            submitButtonsAreDisabled={!this.state.acceptTerms}
+            submitting={submitting}
+            displaySaveForLater={true}
```
Comment (Ronolibert) [inline src/scenes/Moves/Ppm/PaymentReview/index.jsx]: Could also just be this, accomplishes the same affect
```suggestion
            displaySaveForLater
```

File: src/scenes/Legalese/CustomerAgreement.jsx [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const CustomerAgreement = ({ onChange, checked, agreementText }) => {
+const CustomerAgreement = ({ onChange, link, checked, agreementText, className }) => {
```
Comment (Ronolibert) [inline src/scenes/Legalese/CustomerAgreement.jsx]: Could we update the `propTypes` to account for the additional props we added

File: src/scenes/Moves/Ppm/PaymentReview/index.jsx (hunk 3 above)
Comment (mkrump) [reply src/scenes/Moves/Ppm/PaymentReview/index.jsx]: 👍

File: src/scenes/Legalese/CustomerAgreement.jsx (hunk 4 above)
Comment (mkrump) [reply src/scenes/Legalese/CustomerAgreement.jsx]: yep. good call.

File: src/scenes/Moves/Ppm/PaymentReview/index.jsx [hunk 5]
```diff
@@ -12,12 +12,26 @@ import WeightTicketListItem from './WeightTicketListItem';
 import ExpenseTicketListItem from './ExpenseTicketListItem';
 import WizardHeader from '../../WizardHeader';
 import './PaymentReview.css';
+import CustomerAgreement from 'scenes/Legalese/CustomerAgreement';
+import { ppmPaymentLegal } from 'scenes/Legalese/legaleseText';
+import './PaymentReview.css';
```
Comment (jim) [inline src/scenes/Moves/Ppm/PaymentReview/index.jsx]: This looks like a duplicate import.

Comment (mkrump) [reply src/scenes/Moves/Ppm/PaymentReview/index.jsx]: Weird. I feel like that should have got rejected by pre-commit. nice catch!

**Pull request merged**